gui/slick/interfaces/default/config_postProcessing.tmpl

```diff
@@ -70,8 +71,14 @@
                                 <span class="component-title">Process Episode Method:</span>
                                 <span class="component-desc">
                                     <select name="process_method" id="process_method" class="form-control input-sm">
-                                        #set $process_method_text = {'copy': "Copy", 'move': "Move", 'hardlink': "Hard Link", 'symlink' : "Symbolic Link"}
```

Comment:
You don't need to remove this line as it is use to translate the action into a text label, so you don't need L75 and L78 which can be removed.